src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public Ball (String color, int number){

```

Comment:
Please include an access modifier for the constructor. If none is specified, the default package-private is used, which is not recommended. Typically, constructors are `public` unless there's a reason for a different access level.